lib/nostrum/struct/voice_state.ex

```diff
@@ -69,7 +69,9 @@ defmodule Nostrum.Struct.VoiceState do
     end
 
     unless is_nil(v.session_pid) do
-      Session.close_connection(v.session_pid)
+      if Process.alive?(v.session_pid) do
```

Comment:
This is new. For when is this guard introduced? When `ffmpeg` died?

Reply:
This is for when the voice gateway connection may have already closed. This change and changing a call to a cast in the Session module are to prevent an error that can occur when leaving a voice channel.